codeqa.tex

```diff
@@ -44,8 +44,7 @@ \subsubsection{Coding Standards}
 
 Based on the positive  \texttt{flake8} experience on the Python code base, we investigated options for C++.
 Our code layout and formatting rules were updated to be based on the Google C++ Style Guide\cite{googlestyle} with a few minor modifications, such as 4 space indentation and a 110 character line limit to maintain consistency with our Python coding standards.
-This allowed us to easily enforce these rules using the \texttt{clang-format}\cite{clangformat} tool with minimal additional configuration.
-We are currently updating the C++ code incrementally as new changes are made, and we hope to soon be able to run \texttt{clang-format} as part of an automated check when pull requests are created on GitHub.
+This allowed us to easily enforce these rules using the \texttt{clang-format}\cite{clangformat} tool with minimal additional configuration, though developers are also permitted to format their code manually or with any tool consistent with our standards.
```

Comment:
My understanding is that running `clang-format` on large swaths of existing code is explicitly discouraged, and  we have no plans to require all new code to pass through it.  It's essentially an editor-level tool we have worked to allow developers to use if they wish.